ci/cudf_pandas_scripts/fetch_pandas_versions.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import requests
+
+def get_pandas_versions():
+    url = "https://pypi.org/pypi/pandas/json"
+    response = requests.get(url)
+    data = response.json()
+    versions = data['releases'].keys()
+
+    # Filter out pre-releases and major versions
+    minor_versions = sorted(set(
+        '.'.join(version.split('.')[:2]) for version in versions if 'rc' not in version and 'b' not in version
+    ))
+
+    # Remove duplicates and sort
+    minor_versions = sorted(set(minor_versions))
+
+    # Filter for versions starting from 2.0
+    minor_versions = [v for v in minor_versions if v.startswith('2.')]
```

Comment:
Could we have the `2.` be an input to the script so we can update the workflow job instead of this script in the future?

Reply:
Done 👍